conf/default_config.yaml

```diff
@@ -161,23 +161,22 @@ services:
     cycle_timers:
       feed_sync: ${ANCHORE_FEED_SYNC_INTERVAL_SEC} # 6 hours between feed syncs
       feed_sync_checker: 3600 # 1 hour between checks to see if there needs to be a task queued
+      grypedb_sync: 60 # 1 minute between checks to verify local grype-db is up to date
     vulnerabilities:
-      # Available providers are legacy and grype. Legacy provider offers the same matching logic as previous versions <= 0.9.4.
-      # grype is a new provider introduced in 1.0.0 and it uses the grype tool for all things vulnerabilities.
-      # If grype is chose as the provider, uncomment the grypedb data feed in the section below
-      provider: legacy
+      # Available providers are legacy and grype. Legacy provider offers the same matching logic as previous versions of anchore-engine (<= 0.9.4)
+      # grype is a new provider that was introduced in 1.0.0, it uses the grype tool for all things vulnerabilities
+      provider: ${ANCHORE_VULNERABILITIES_PROVIDER}
       sync:
         enabled: ${ANCHORE_FEEDS_ENABLED}
         ssl_verify: ${ANCHORE_FEEDS_SSL_VERIFY}
         connection_timeout_seconds: 3
         read_timeout_seconds: 60
         data:
-          # Uncomment if the provider is set to grype. If packages feed was enabled and necessary leave it uncommented.
-          # All remaining feeds can be commented out
-          #grypedb:
-          #  enabled: true
-          #  url: ${ANCHORE_GRYPE_DB_URL}
-          # The following feeds except packages are required only for legacy provider. Comment them out if the provider is set to grype
+          # grypedb feed is synced if the provider is set to grype. All the remaining feeds except for packages are ignored even if they are enabled
+          grypedb:
```

Comment:
you uncommented this. intentional?

Reply:
Yep, allowed feeds are maintained in the provider, shouldn't matter what the config has

Reply:
I think it is tied to the provider now where it will do an either or based upon what provider is set. See providers [default config](https://github.com/anchore/anchore-engine/blob/55bb293f9d8edb54f82c6825d36f30238fb029bf/anchore_engine/services/policy_engine/engine/vulns/providers.py#L748-L748) and how that it is used [in the config](https://github.com/anchore/anchore-engine/blob/55bb293f9d8edb54f82c6825d36f30238fb029bf/anchore_engine/services/policy_engine/engine/feeds/config.py#L124-L147)